.github/workflows/build-java-module.yaml

```diff
@@ -185,103 +167,25 @@ jobs:
             -DIMAGE_NAME="${ECR_REGISTRY}/${ECR_REPOSITORY_PREFIX}" \
             -DTAG_COMMIT=${{ env.HAWK_IMAGE_TAG }}
 
-  vulnerability-check-init:
```

Comment:
the vulnerability change was deleted from here (to reduce the maintenance effort) because it's already in the global pipeline that builds all features anyway.